content/help/ops/telemetry/missing-metrics/index.md

```diff
@@ -69,28 +70,39 @@ For reference, please consult the [default rules for Prometheus]({{< github_file
     istio-system   handler   13d
     {{< /text >}}
 
+    Depending on whether or not your install of Istio was a fresh install or upgrade, you may also need to issue the following command:
+
+    {{<text bash>}}
```

Comment:
My silly linting script insists on having spaces in there:

{{< text bash >}}

Reply:
ok. makes sense. i was rushing through fixes. updated.